src/rust/plugin-registry/src/nomad/client.rs

```diff
@@ -137,6 +139,24 @@ impl NomadClient {
         .await
         .map_err(NomadClientError::from)
     }
+
+    #[tracing::instrument(skip(self, job_name, namespace), err)]
+    pub async fn delete_job(
+        &self,
+        job_name: String,
+        namespace: Option<String>,
+    ) -> Result<models::JobDeregisterResponse, NomadClientError> {
+        jobs_api::delete_job(
+            &self.internal_config,
+            jobs_api::DeleteJobParams {
+                namespace: namespace.clone(),
+                job_name: job_name.to_owned(),
```

Comment:
I think both `.clone()` and `.to_owned()` are unnecessary here

Reply:
fixed